packages/cadl-rlc-emitter/test/integration/generated/models/readonlyProperties/README.md

```diff
@@ -0,0 +1,56 @@
+# ReadonlyProperties client library for JavaScript
+
+Readonly Property Test Service
+
+**Please rely heavily on our [REST client docs](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/rest-clients.md) to use this library**
```

Comment:
if it's generate rlc for mgmt plane, please consider the comment here https://github.com/Azure/azure-sdk-for-js/pull/22044#discussion_r893498739

Reply:
Currently both of rlc and data-plane share the same link, do you know which link should be used when data-plane

Reply:
updated